src/cli/run.js

```diff
@@ -1,20 +1,20 @@
 import forkTerminal from '@shoutem/fork-terminal';
 import path from 'path';
 
-export const description = 'Run shoutem application on using Shoutem preview app';
+export const description = 'Runs Shoutem app on using Shoutem preview app';
```

Comment:
I'm not sure what this sentence should say, but it sounds wrong.

Reply:
It's the legacy development flow, it runs your local extensions inside something like our platform repository and lets you preview it on the Shoutem preview app.

I'll put:
"Runs and previews your local code with Shoutem preview app." 
But it doesn't really matter because this flow hasn't been used for years and probably can't even be used anymore since it only supports platform 0.58 or lower, something like that.

Reply:
We should check this (latest supported platform) and add a note about that. Please create a task for this, we don't have to do it right away.